packages/manager/modules/pci/src/projects/projects.routing.js

```diff
@@ -23,11 +20,38 @@ export default /* @ngInject */ ($stateProvider) => {
       },
       resolve: {
         breadcrumb: /* @ngInject */ () => null,
+        confirmDeletion: /* @ngInject */ $state => project => $state.go('pci.projects.remove', { projectId: project.project_id }),
+        defaultProject: /* @ngInject */
+          PciProjectsService => PciProjectsService.getDefaultProject(),
+        getProject: /* @ngInject */ OvhApiCloudProject => project => OvhApiCloudProject
+          .v6()
+          .get({ serviceName: project.serviceName })
+          .$promise
+          .then(projectDetails => new Project(projectDetails)),
```

Comment:
I don't know if you share the same feeling as mine but it means that all
properties returned by the API `cloud.Project` will be accessible on the Front.

Because in your class constructor you are using ` Object.assign(this, project);`.

But in some cases, some of these properties aren't relevant (or they act more
as a private one).

That's why the getters can provide a more granular control layer on what can
or cannot be accessible on the Front.

Any feedback is well appreciate.

Thanks!

Reply:
I don't understand your point, because using the class or directly `projectDetails` will expose the same API payload

Reply:
Oh! nevermind then!

I asked myself if there was a way to make a distinction between public and private properties which is not the case as of now.